cmd/utils/flags.go

```diff
@@ -903,6 +903,21 @@ var (
 		Usage: "When enabled a RPC node can use the L2 to build the InfoTree.",
 		Value: false,
 	}
+	Hardfork = cli.StringFlag{
+		Name:  "zkevm.hardfork",
+		Usage: "Hermez | Ethereum. Default hermez. Ethereum enabled the traditional hardfork selection system.",
+		Value: "hermez",
+	}
+	Commitment = cli.StringFlag{
+		Name:  "zkevm.commitment",
+		Usage: "Dictates the state commitment trie: SMT | PMT | SMTV2",
```

Comment:
Make this consistent with the above Usage: "Values { smt | pmt }. Default, smt. ..."